src/cbmc_starter_kit/setup_ci.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env python3
+
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+
+"""Set up the AWS infrastructure to run proofs in CI."""
+
+import logging
+import re
+import shutil
+from argparse import RawDescriptionHelpFormatter
+
+
+from cbmc_starter_kit import arguments, repository, util
+
+################################################################
+
+def parse_arguments():
+    """Parse arguments for cbmc-starter-kit-setup-ci command"""
+    desc = """
+    Adds a worflow and other resource files, such that CBMC proofs are executed
+    in GitHub Actions as part of CI.
+
+    The most recently released and available versions of CBMC, CBMC viewer,
+    Litani, kissat and cadical will be used by default. In the cases of CBMC,
+    CBMC viewer and Litani, semantic versions can be specified (e.g. '5.71.0'
+    for CBMC, '3.6' for CBMC viewer, '1.27.0' for Litani). In the cases of
+    kissat and cadical, any released tag can be specified (e.g. 'rel-3.0.0' and
+    'rel-1.5.3' respectively).
+
+    To find semantic versions for these tools or tags for the solvers, visit:
+    - https://github.com/diffblue/cbmc/releases
+    - https://github.com/model-checking/cbmc-viewer/releases
+    - https://github.com/awslabs/aws-build-accumulator/releases
+    - https://github.com/arminbiere/kissat/tags
+    - https://github.com/arminbiere/cadical/tags
+    """
+    options = [{
+        'flag': '--github-actions-runner',
+        'required': True,
```

Comment:
`ubuntu-20.04` is not a large runner though. It's the standard GitHub-hosted runner.

Reply:
> Why isn't ubuntu-20.04 the default

This default will not work for most CBMC proof projects. When I added this flag, I wanted users to be explicit about the name/type of Ubuntu-20.04 runner that will be used during execution of proofs.

Reply:
I prefer to keep this as is for now.

Reply:
How would a user know what runner to use if they're setting this up for the first time?

If you were running this script on a new codebase, which runner would you use? Personally I would want to see if it worked on the standard runner before even bothering to try setting up a Large Runner. I think `ubuntu-20.04` should be the default.

I don't think any of these flags should be required because there's no way for a user to know what value to use without trying.

> ubuntu-20.04 is not a large runner though. It's the standard GitHub-hosted runner.

Right, so maybe the flag name I chose is confusing. But the intent is what you describe: the default is to use the standard runner, if you want to use a large runner, you pass `--large-runner`.

Reply:
I will use `ubuntu-20.04` as the default GitHub Actions runner.